### ISSUE TITLE
darwin x86_64 syscall numbering is incomplete

### ISSUE THREAD

**Issue by lunixbochs** [lunixbochs/lib43#21]
the constants have `(2 << 24)` blindly added, which is only valid for unix (not mach, etc)

see https://github.com/lunixbochs/usercorn/issues/91

**Comment by ghost** [comment]
Right. Are you thinking using an enum to specify this behavior and just use it to repress possible bit shift values?
